app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+    belongs_to :author, class_name: 'User'
+    has_many :comments
+    has_many :likes
+
+    after_save :post_counter_update
+
+    def most_recent_comments
+      comments.order(created_at: :desc).limit(5)
+    end
+  
+    private
```

Comment:
**Kudos for putting post_counter_update**

- [ ] I suggest the same to other update methods by putting them after `private`